cg/cli/upload/vogue.py

```diff
@@ -199,4 +244,4 @@ def _get_analysis_workflow_details(context, case_name):
         workflow_name = family_obj.analyses[0].pipeline
         workflow_version = family_obj.analyses[0].pipeline_version
 
-    return workflow_name, workflow_version
+    return workflow_name.lower(), workflow_version
```

Comment:
Same here, no need for sending the whole context. Just pick out the store before.

Reply:
I made changes according to comment.